recipes/libnice/all/conanfile.py

```diff
@@ -59,16 +60,16 @@ def validate(self):
                 "-o glib/*:shared=True with static runtime is not supported")
 
     def requirements(self):
-        self.requires("glib/2.75.2")
+        self.requires("glib/2.77.1", transitive_headers=True, transitive_libs=True)
         if self.options.crypto_library == "openssl":
-            self.requires("openssl/1.1.1t")
+            self.requires("openssl/[>=1.1 <4]")
         if self.options.with_gstreamer:
-            self.requires("gstreamer/1.19.2")
+            self.requires("gstreamer/1.22.3")
 
     def build_requirements(self):
-        self.tool_requires("meson/1.0.0")
-        self.tool_requires("pkgconf/1.9.3")
-        self.tool_requires("glib/2.75.2")  # for glib-mkenums
+        self.tool_requires("meson/1.2.1")
+        self.tool_requires("pkgconf/1.9.5")
+        self.tool_requires("glib/<host_version>")  # for glib-mkenums
```

Comment:
Is this recipe missing a conan min version?

Reply:
Good catch! I'll update it, thank you!

Reply:
Done in the commit [9cba47e](https://github.com/conan-io/conan-center-index/pull/19037/commits/9cba47eec1a669dfdd0068185538436310d92d8a)